client/components/close-on-escape/test/index.jsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import React from 'react';
+import ReactDom, { unmountComponentAtNode } from 'react-dom';
+import { spy } from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import useFakeDom from 'test/helpers/use-fake-dom';
+import CloseOnEscape from '../';
+
+const simulateEscapeKeydown = () => (
```

Comment:
Might it be worth extracting this in to a test utility? Possibly at a later date when a similar case arises where a keydown is triggered on the document elsewhere.